e2e/scenario_test/settings/not_set/basal_rates/settings.no_basal_rates.spec.js

```diff
@@ -0,0 +1,51 @@
+const { setup, pump, cgm, match, settings } = require('../../../../../src/index');
+
+describe('Pump Settings', () => {
+    beforeAll(async () => {
+        await setup.LaunchLoop();
+        await setup.LoadScenarios(device.id);
+        await cgm.Add();
+        await pump.Add();
+    });
+    describe('Closed loop is not allowed', () => {
+        describe('When basal rates are not set', () => {
+            beforeAll(async () => {
+                await setup.LoadScenario('flat_cgm');
```

Comment:
[?] How does the `LoadScenario` by name interact with the `LoadScenario` by device id in the `beforeAll`?

Reply:
`LoadScenarios` by device id will load all available scenerio's onto the device by copying them into place. `LoadScenario` will load the specific scenario